src/Task/style.css

```diff
@@ -24,6 +24,17 @@
             fill: var(--gray);
         }
     }
+    &.is-extra{
+        & .ola_task-title-text {
+            padding-right: var(--size-6);
+            flex: 1 1 auto;
+            @media (--to-width-1) {
+                text-overflow: ellipsis;
+                overflow: hidden;
+                white-space: nowrap;
+            }
+        }
+    }
```

Comment:
Instead padding-right of `.ola_task-title-text` (now renamed to `ola_task-title`, we can add margin-left to `.ola_task-extra`, so we can remove the `is-extra` modifier (the other css properties can be added no matter if we have extra or not).

Reply:
done